utils/resultstable.go

```diff
@@ -930,15 +930,14 @@ func convertCves(cves []services.Cve) []formats.CveRow {
 
 // If at least one cve is applicable - final value is applicable
 // Else if at least one cve is undetermined - final value is undetermined
+// Else if all cves are not covered -> final value is not covered
 // Else (case when all cves aren't applicable) -> final value is not applicable
 func getApplicableCveStatus(entitledForJas bool, applicabilityScanResults []*sarif.Run, cves []formats.CveRow) ApplicabilityStatus {
-	if !entitledForJas || len(applicabilityScanResults) == 0 {
+	if !entitledForJas || len(applicabilityScanResults) == 0 || len(cves) == 0 {
 		return NotScanned
 	}
```

Comment:
```suggestion
	if !entitledForJas || len(applicabilityScanResults) == 0 {
		return NotScanned
	}
    if len(cves) == 0 {
		return NotCovered
	}
```
it should be the default when JAS is enabled, we still show the column, it will be the only row with empty values...

Reply:
In case we have 0 cves it means that Xray didn't find any cves for this vulnerability / violation.
I think it's different than not covered - which means that the scanner did run but there is no rule for this cve.